src/components/Overlay.tsx

```diff
@@ -78,13 +78,10 @@ const Overlay = ({ qboard }: { qboard: QBoard }): JSX.Element => {
   return (
     <>
       <VirtualFileInput
-        acceptFiles={async (file) =>
-          qboard.history.execute((await qboard.files.acceptFile(file)).history)
-        }
+        acceptFiles={qboard.files.acceptFile}
         captureRef={(ref) => {
           qboard.globalState.fileInputRef = ref;
         }}
-        multiple
```

Comment:
#182?

Reply:
Yeah, this is done in 0d814b5154649dfb92a5456e3f5a2af34af20c9b to at least make the input consistent with the processing.

https://github.com/cjquines/qboard/issues/182#issuecomment-860254183 introduces a different solution, but that can still be done afterwards